src/domain/transfer/index.js

```diff
@@ -176,30 +154,29 @@ const getTransferById = async (headers, params, span) => {
 * @returns {boolean} Returns true on successful publishing of message to kafka, throws error on failures
 */
 const transferError = async (headers, dataUri, payload, params, span) => {
-  Logger.isDebugEnabled && Logger.debug(`domain::transfer::abort::start(${params.id}, ${JSON.stringify(headers)}, ${JSON.stringify(payload)})`)
+  // todo: determine fxTransfer
+  const logPrefix = 'domain::transfer::abort'
+
+  logger.debug(`${logPrefix}::start(${params.id}, ${JSON.stringify(headers)}, ${JSON.stringify(payload)})`)
   try {
-    const state = StreamingProtocol.createEventState(generalEnum.Events.EventStatus.SUCCESS.status, generalEnum.Events.EventStatus.SUCCESS.code, generalEnum.Events.EventStatus.SUCCESS.description)
-    const event = StreamingProtocol.createEventMetadata(generalEnum.Events.Event.Type.FULFIL, generalEnum.Events.Event.Action.ABORT, state)
+    const state = dto.eventStateDto()
```

Comment:
Don't you want to pass any arguments in this?

Reply:
Based on current impl. `state` is always the same:
```
const state = StreamingProtocol.createEventState(generalEnum.Events.EventStatus.SUCCESS.status, generalEnum.Events.EventStatus.SUCCESS.code, generalEnum.Events.EventStatus.SUCCESS.description)
```